plugins/system/accessibility/accessibility.php

```diff
@@ -68,6 +68,9 @@ public function onBeforeCompileHead()
         // Detect the current active language
         $lang = Factory::getLanguage()->getTag();
 
+        //detect the position
+        $position = $this->params->get('position','bottom_left');
```

Comment:
```suggestion
        $position = $this->params->get('position', 'bottom_left');
```
https://ci.joomla.org/joomla/joomla-cms/59354/1/8

Reply:
Corrected